src/lib/sections/Navigation/Link/Link.stories.tsx

```diff
@@ -27,9 +27,16 @@ const meta: Meta<typeof Navigation.Link> = {
 };
 
 export default meta;
-export const Example = { args: { to: "#", current: "page", children: (
-  <>
-    <Navigation.Icon name="code" />
-    <Navigation.Label>Settings</Navigation.Label>
-  </>
-)}};
\ No newline at end of file
+export const Example = {
```

Comment:
It would be great to have a second example here with the link displayed as a Button, something like this:

```ts
export const ButtonExample = {
  args: {
    as: Button, // import this from @canonical/react-components
    appearance: "link",
    children: (
      <>
        <Navigation.Label>Log out</Navigation.Label>
      </>
    ),
  },
}
```